smithy-aws-protocol-tests/model/restJson1/services/glacier.smithy

```diff
@@ -0,0 +1,168 @@
+$version: "1.0"
+
+namespace com.amazonaws.glacier
+
+use aws.api#service
+use aws.auth#sigv4
+use aws.protocols#restJson1
+use smithy.test#httpRequestTests
+
+@service(
+    sdkId: "Glacier",
+    arnNamespace: "glacier",
+    cloudFormationName: "Glacier",
+    cloudTrailEventSource: "glacier.amazonaws.com",
+    endpointPrefix: "glacier",
+)
+@sigv4(
+    name: "glacier",
+)
+@restJson1
+@title("Amazon Glacier")
+@xmlNamespace(
+    uri: "http://glacier.amazonaws.com/doc/2012-06-01/",
+)
+service Glacier {
+    version: "2012-06-01",
+    operations: [
+        UploadArchive,
```

Comment:
This should also include `UploadPart`, to validate that the checksum customizations are made to both operations.